tests/phpunit/includes/object-cache.php

```diff
@@ -745,6 +745,17 @@ function wp_cache_init() {
 	$wp_object_cache = new WP_Object_Cache();
 }
 
+/**
+ * Whether the object cache implementation supports flushing individual cache groups.
+ *
+ * @since 6.1.0
+ *
+ * @return bool True if group flushing is supported, false otherwise.
+ */
+function wp_cache_supports_group_flush() {
+	return false;
```

Comment:
should this not be true, built in cache does support group flushes.

Reply:
This is referring to the PHPUnit `WP_Object_Cache` class that uses Memcached, which currently does not support group flushing.

I'd also like to suggest not bothering to add support to this test class, since it's not used and testing unused code is useless.